modules/cudl-data-processing/locals.tf

```diff
@@ -9,12 +12,20 @@ locals {
       filter_prefix = notification.filter_prefix
       filter_suffix = notification.filter_suffix
       queue_names   = [for subscription in notification.subscriptions : subscription.queue_name]
-    } if contains(var.source-bucket-names, notification.bucket_name)
+    } if contains(local.transform-lambda-bucket-names, notification.bucket_name)
   }
   source_bucket_sqs_notifications = {
-    for bucket in toset(var.source-bucket-names) : bucket => [
+    for bucket in local.transform-lambda-bucket-names : bucket => [
       for notification in var.source-bucket-sqs-notifications : notification if notification.bucket_name == bucket
     ]
   }
   transform_lambda_queues = toset([for lambda in var.transform-lambda-information : lambda.queue_name])
+
+
+
+  transform-lambda-buckets = {
+    for bucket in toset([aws_s3_bucket.source-bucket, aws_s3_bucket.dest-bucket]) :
```

Comment:
This for loop could refer to `local.transform-lambda-bucket-names`

Reply:
I'm not sure how we may refactor this to get the required map of s3 bucket objects from the string name value for these objects in Terraform.  If we could get a list of initiated s3 bucket objects that would be trivial but I don't think that's possible in terraform. Anyway I will leave as is, for now as a straight swap for the names list wouldn't work here.